src/validator/diagnostics.ts

```diff
@@ -32,16 +32,8 @@ import {
 
 export const TABNINE_DIAGNOSTIC_CODE = "TabNine";
 
-const thresholdMap: Map<string, number> = new Map([
-  ["Low", 35],
-  ["Medium", 65],
-  ["High", 85],
-]);
-const reverseThresholdMap: Map<number, string> = new Map();
-thresholdMap.forEach((value, key) => reverseThresholdMap.set(value, key));
-
-let BACKGROUND_THRESHOLD = thresholdMap.get("Medium");
-const PASTE_THRESHOLD = 1;
+let BACKGROUND_THRESHOLD = "Medium";
```

Comment:
const

Reply:
The threshold may be changed, thus the `let`